tools/ClassGen/NodeGen.cpp

```diff
@@ -684,12 +684,14 @@ int main(int argc, char **argv) {
   BB.newNode("Reshape")
       .addInput("Input")
       .addMember(MemberType::VectorSizeT, "Dims")
+      .addMember(MemberType::String, "Layout")
```

Comment:
Our plan is to move the layout information from nodes to Glow types. this can be done in a follow-up commit, even as a first task for someone motivated, during an operator's construction it will propagate the layout string (a pointer to global hash of strings to save on size). so we will not add a member called layout.
said follow-up is straightforward / just a lot of manual labor if we are fine with the design - no point in something fancy in here :)